_posts/en/newsletters/2023-04-12-newsletter.md

```diff
@@ -0,0 +1,184 @@
+---
+title: 'Bitcoin Optech Newsletter #246'
+permalink: /en/newsletters/2023/04/12/
+name: 2023-04-12-newsletter
+slug: 2023-04-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Splicing specification discussions:** several LN developers posted
+  to the Lightning-Dev mailing list this week about ongoing work on the
+  [draft specification][bolts #863] for [splicing][topic splicing],
+  which allows some funds in an offchain LN channel to be spent onchain
+  (splice-out) or for onchain funds to be added to an offchain channel
+  (splice-in).  The channel can continue to operate while an onchain
+  splicing transaction is waiting for sufficient confirmations.
+
+  {:.center}
+  ![Splicing transaction flow](/img/posts/2023-04-splicing1.dot.png)
+
+  Discussions this week included:
+
+  - *Which commitment signatures to send:* when a splice is created, a
+    node will hold parallel commitment transactions, one that spends
+    from the original funding output and one that spends from each new
+    funding output for all pending splices.  As of this writing, the
+    current draft specification involves sending signatures for every
+    commitment transaction that could possibly need to be confirmed,
+    even for commitment transactions whose signatures have previously
+    been sent.  As Lisa Neigut [explains][neigut splice], this allows reusing the
+    workflow already used in LN for updating a commitment transaction.
+    However, it's requires extra work and bandwidth, so the alternative
+    of only sending signatures for any new commitment transactions is
+    being contemplated.
+
+  - *Relative amounts and zero-conf splices:* Bastien Teinturier
+    [posted][teinturier splice] about several proposed specification
+    updates.  In addition to the aforementioned commitment signatures
+    change, he also recommends that splice proposals use relative
+    amounts, e.g. "200,000 sats" indicates Alice will splice in that
+    amount and "-50,000 sats" indicates she wants to splice out that
+    amount.  He also mentions, but does not go into detail, about a
+    concern involving zero-conf splicing.
+
+- **Proposed BIP for transaction terminology:** Mark "Murch" Erhardt
+  [posted][erhardt terms] the draft of an [information BIP][terms bip]
```

Comment:
nit: seems "informational" names the type of bip? https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-types
```suggestion
  [posted][erhardt terms] the draft of an [informational BIP][terms bip]
```